sys/newlib/syscalls.c

```diff
@@ -71,8 +72,8 @@ void _fini(void)
  */
 void _exit(int n)
 {
-    printf("#! exit %i: resetting\n", n);
-    reboot(n);
+    LOG_INFO("#! exit %i: resetting\n", n);
```

Comment:
Isn't this message likely to get cut if logging to UART because of the removed delay loop?